layouts/partials/head.html

```diff
@@ -11,11 +11,21 @@
 <meta name="viewport" content="width=device-width, initial-scale=1.0" />
 <link rel="alternate" type="application/rss+xml" href="{{ .Site.BaseURL }}index.xml" title="{{ .Site.Title }}">
 <!-- Stylesheets -->
-<link id="dark-mode-theme" rel="stylesheet" href="{{ .Site.BaseURL }}css/dark.css" />
 <link rel="stylesheet" href="{{ .Site.BaseURL }}fontawesome/css/all.min.css" />
 {{ range .Site.Params.customCSS }}
 <link rel="stylesheet" href="{{ . | absURL }}" />
 {{ end }}
+<link id="dark-mode-theme" rel="stylesheet" href="{{ .Site.BaseURL }}css/dark.css" />
+<!-- Load theme mode from browser cache -->
+<script>
+	var darkTheme = document.getElementById("dark-mode-theme");
+	var storedTheme = localStorage.getItem("dark-mode-storage");
+	if ( storedTheme === "dark") {
+		darkTheme.disabled = false;
+    } else if (storedTheme === "light") {
+		darkTheme.disabled = true;
+    }
+</script>
```

Comment:
Please remove unnecessary whitespace in `if` clause and fix indent. (I know I should prepare linter...)

Reply:
Done!